tests/analysis/test_peak_detection.py

```diff
@@ -363,6 +363,20 @@ def test_find_twitch_indices__returns_correct_values_with_data_that_ends_in_vall
     assert actual[3][SUBSEQUENT_VALLEY_INDEX_UUID] == 4
 
 
+def test_peak_finding_return_exeption_when_peaks_are_missing():
+    peak_finding_folder = os.path.join(
+        get_current_file_abs_directory(), os.pardir, "data_files", "peak_finding"
+    )
+
+    test_file_path = os.path.join(peak_finding_folder, "waveforms", "waveform_3.npy")
```

Comment:
These test files we use should have descriptive names so we can at least get some idea of why they are included